checkov/common/runners/base_runner.py

```diff
@@ -3,12 +3,14 @@
 import re
 from abc import ABC, abstractmethod
 from typing import List, Dict, Optional, Any
+from distutils.util import strtobool
```

Comment:
Just a heads up, `distutils` will be deprecated with Python 3.10.
https://www.python.org/dev/peps/pep-0632/#specification

Reply:
good point. copying the specific method to our code base